requirements.txt

```diff
@@ -27,3 +27,4 @@ https://files.pythonhosted.org/packages/16/4a/b085a04d6dad79aa5c00c65c9b2bbcb2c6
 lxml==4.3.0; sys_platform == 'win32'
 GitPython==2.1.11
 launchpadlib==1.10.6
+yamlordereddictloader==0.4.0
```

Comment:
why this and not enhance snapcraft.yaml_utils?

Reply:
Oh that would be much more appropriate :)